README.md

```diff
@@ -13,31 +13,32 @@ Powertools for AWS Lambda (Java) is a developer toolkit to implement Serverless
 
 Powertools for AWS Lambda (Java) is available in Maven Central. You can use your favourite dependency management tool to install it
 
-* [maven](https://maven.apache.org/):
+#### Maven:
 ```xml
 <dependencies>
     ...
     <dependency>
         <groupId>software.amazon.lambda</groupId>
         <artifactId>powertools-tracing</artifactId>
-        <version>1.17.0-SNAPSHOT</version>
+        <version>1.16.0</version>
```

Comment:
Why are we changing the version in this PR?
is this something we need to flip back after the release process?

Reply:
We move to next snapshot after a release. At the release (1.17.0), we'll move to 1.18.0-SNAPSHOT to avoid working on an existing version and have strange behaviours.